packages/app/src/components/VisualizationOptions/Options/HideTitle.js

```diff
@@ -26,9 +26,12 @@ class HideTitle extends Component {
         this.state = props.value ? { value: props.value } : this.defaultState
     }
 
-    onChange = ({ value }) => {
+    onRadioGroupChange = ({ value }) => {
         this.setState({ value })
-        this.props.onChange(value === 'NONE')
+        this.props.onChange({
+            hideTitle: value === 'NONE',
```

Comment:
Can all of these strings (`NONE`, `AUTO` etc) be standardised as `const` instead?